builds/misc/images-release.yaml

```diff
@@ -32,11 +34,15 @@ jobs:
       - template: ../templates/dotnet3-globaljson.yaml # use dotnet 3 as primary install for build
       - bash: "cargo install --git https://github.com/arsing/cross.git --branch set-path"
         displayName: "Install cross (fork with docker fix)"
+      # Build
       - task: ShellScript@2
         displayName: "Build Azure-IoT-Edge-Core"
         inputs: 
           args: "-c Release"
           scriptPath: scripts/linux/buildBranch.sh
+      - template: templates/build-broker-watchdog.yaml
```

Comment:
q: we used templates to share some steps between jobs, but I didn't find other places with this template. do you think it is still worthwhile to decompose it with these 2 templates?

Reply:
We also use this in images.yaml